sql/7_0_0-to-7_0_1_upgrade.sql

```diff
@@ -105,3 +105,26 @@
 --    desc: Change Layout edit options.
 --    arguments: mode(add or remove) layout_form_id the_edit_option comma_separated_list_of_field_ids
 
+#IfNotTable questionnaire_repository
+CREATE TABLE `questionnaire_repository` (
+    `id` bigint(21) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `uuid` binary(16) DEFAULT NULL,
+    `questionnaire_id` varchar(255) DEFAULT NULL,
+    `provider` int(11) UNSIGNED DEFAULT NULL,
+    `version` int(11) NOT NULL DEFAULT 1,
+    `created_date` datetime DEFAULT current_timestamp(),
+    `modified_date` datetime DEFAULT current_timestamp(),
+    `name` varchar(255) DEFAULT NULL,
+    `type` varchar(63) NOT NULL DEFAULT 'Questionnaire',
+    `profile` varchar(255) DEFAULT NULL,
+    `active` tinyint(2) NOT NULL DEFAULT 1,
+    `status` varchar(31) DEFAULT NULL,
+    `source_url` text,
+    `code` varchar(255) DEFAULT NULL,
```

Comment:
Will this be a LOINC code?  What goes here?

Reply:
It could be a loinc code or whatever code the questionnaire sets in resource. So I guess I need to add a code system url from questionnaire
`

`

Reply:
hmm on second thought the code is only there for searching as meta and is not pertinent to form render or anything else. The code is parsed from the resource anyway. The value sets are within the resource.